ee/cis/win-10/cis-policy-queries.yml

```diff
@@ -5641,6 +5641,88 @@ spec:
 ---
 apiVersion: v1
 kind: policy
+spec:
+  name: >
+    CIS - Ensure 'Enable news and interests on the taskbar' is set to 'Disabled'
+  platforms: win10
+  platform: windows
+  description: |
+    This policy setting specifies whether the news and interests feature is allowed on the device.
+    The recommended state for this setting is: Disabled.
```

Comment:
I think we removed the recommended state line from all of our policies because it's stated in the name/title as well as the resolution

Reply:
It's in 58/311 policies we committed so I don't think it _really_ matters.